docs/guides/platform/manager/service-transfers/index.md

```diff
@@ -0,0 +1,143 @@
+---
+slug: service-transfers
+author:
+  name: Linode
+  email: docs@linode.com
+description: 'Send your Linode services to another owner conveniently with Service Transfers.'
```

Comment:
I know we probably want to avoid the double usage of "transfer" in a sentence, but I think transfer is better than send here, as it's more descriptive.